Permissioned DEX XLS.md

```diff
@@ -0,0 +1,467 @@
+* Why all this isn't enough - DEX activity
+
+### 1.2. Terminology
+
+* **Offer Crossing**: Two offers **cross** if one is selling a token at a price that's actually lower than the price the other is offering to sell it at.
+* **Offer Filling**: Two offers **fill** each other if the trade executes and the sale goes through. Offers can be partially filled, based on the flags (settings) of the offers.
+* **Domain**: A collection of rules indicating what accounts and trades may be a part of it. This spec includes credential-gating and token-gating, but more options could be added in the future.
+* **Domain Rules**: The set of rules that govern a domain, i.e. the credentials and tokens it accepts.
+* **Domain Owner**: The account that created a domain, and is the only one that can modify its rules or delete it.
+* **Domain Member**: An account that satisfies the rules of the domain (i.e. has one of the credentials that are accepted by the domain). There is no explicit joining step; as long as the account has a valid credential, it is a member.
+* **Permissioned DEX**: The subset of the DEX that operates within the rules of a specific domain.
+* **Open DEX**: The unpermissioned DEX that has no restrictions.
+* **Permissioned Offer/Payment**: An offer/cross-currency payment that can only be filled by offers that are a part of a specific domain.
+* **Open Offer/Payment** or **Unpermissioned Offer/Payment**: An offer/cross-currency payment that is able to trade on the open DEX or potentially in permissioned DEXes, but doesn't have any restrictions itself.
+* **Valid Domain Offer/Payment**: An offer/cross-currency payment that satisfies the rules of a domain (i.e. the account is a domain member, and the tokens in the offer are accepted). This can be a permissioned _or_ open offer.
+
+### 1.3. Basic Flow
+
+#### 1.3.1. Initial setup
+* Owen, a domain owner, creates his domain with a set of KYC credentials and allowed tokens (including a certain USD and EUR).
+* Tracy, a trader, is operating in Owen's regulatory environment and has strict regulatory requirements - she cannot receive liquidity from non-KYCed accounts. She has one of Owen's accepted KYC credentials, and will only be placing permissioned offers.
+* Marko, a market maker, wants to arbitrage offers in Owen's domain, as there is often a significant price difference inside and outside. He obtains one of the KYC credentials that Owen's domain accepts. However, he will not be placing permissioned offers, only open offers, to access as much liquidity as possible.
+
+#### 1.3.2. Trading Scenario 1
+
+* Tracy places a permissioned offer on the XRP-USD orderbook. There are no other offers on that orderbook that are a part of the domain.
+* Marko notices that there is a significant price difference between Tracy's offer and the rest of the orderbook. He now submits an open offer to cross and fill Tracy's offer.
+
+#### 1.3.3. Trading Scenario 2
+* Marko has placed open offers on the XRP-EUR orderbook.
```

Comment:
Marko's open offer can cross any open offer. If it's a valid domain offer then can it cross the domain offers? If yes then how does it work? Each domain offer is going to have it's own order book. We need to know the DomainID to access those order books. The applies to the payments.

Reply:
With the spec change to have separate orderbooks, this is no longer relevant.